getting-started/web-app.md

```diff
@@ -0,0 +1,325 @@
+
+1. Open Visual Studio and select **Create a new project**.
+
+1. Select the [**Blazor Web App**](https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app) project type, enter a name for your project, and then click **Next**.
+
+1. Select the .NET 8 framework, the desired [interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and interactivity location (per page or global).
+
+1. Click **Create**.
+
+@[template](/_contentTemplates/common/get-started.md#add-nuget-feed)
+
+## Step 3: Install the Telerik UI for Blazor Components
+
+1. Right-click  the Blazor Server project in the solution and select **Manage NuGet Packages**.
+
+1. Install the Telerik Blazor NuGet package:
+
+  1. Select the `telerik.com` **Package source** that you [added earlier](#step-2-add-the-telerik-nuget-feed-to-visual-studio). As this is a private NuGet feed, you must authenticate with your [Telerik account](https://www.telerik.com/account/) user name and password.
+  1. Select the **Browse** tab, find the NuGet package, and click **Install**. Commercial license holders should install `Telerik.UI.for.Blazor`. Trial users should install `Telerik.UI.for.Blazor.Trial`.
+
+## Step 4: Enable the Blazor UI Components
+
+To enable the Telerik UI for Blazor components, you must add several client-side dependencies to the application:
+
+### 4.1. Add the Telerik UI for Blazor Client Assets
+
+1\. Add the `telerik-blazor.js` file to your `App.razor` file as a [static asset]({%slug getting-started/what-you-need%}#using-static-assets) or from a [CDN]({%slug getting-started/what-you-need%}#using-cdn).
+
+**HTML**
+
+@[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
+
+2\. To select the appearance and color scheme for the Telerik Blazor components, add the [theme stylesheet]({%slug general-information/themes%}) in your `App.razor` file. Reference it as a [static asset]({%slug general-information/themes%}#static-assets) or from a [CDN]({%slug general-information/themes%}#cdn).
+
+3\. (optional - applicable if you are using static assets) To enable the use of static assets in your project, add the `app.UseStaticFiles();` line to the `Program.cs` of your Blazor Server project (by default, this line is already present).
+
+**C#**
+@[template](/_contentTemplates/common/js-interop-file.md#enable-static-assets-snippet)
+
+###  4.2. Include @using Statements
+
+In the `~/_Imports.razor` file, add the `@using` directives below. This configures the project to recognize the Telerik components in all files. You can register one or both icon namespaces, depending on the [icon type you will be using]({%slug general-information/font-icons%}).
+
+**_Imports.razor**
+    
+    @using Telerik.Blazor
+    @using Telerik.Blazor.Components
+    @using Telerik.FontIcons
+    @using Telerik.SvgIcons
+
+###  4.3. Add the TelerikRootComponent
```

Comment:
```suggestion
###  4.3. Add the Telerik Root Component
```

Reply:
This is the name of the component and we reference it like so in the entire documentation.